include/SFML/Audio/SoundRecorder.hpp

```diff
@@ -145,6 +145,34 @@ class SFML_AUDIO_API SoundRecorder : AlResource
     const std::string& getDevice() const;
 
     ////////////////////////////////////////////////////////////
+    /// \brief Set the channel count of the audio capture device
+    ///
+    /// This method allows you to set the channel count of the
+    /// recording. Currently only 16-bit mono and 16-bit stereo
+    /// are supported.
+    ///
+    /// \param channelCount Integer specifying the channel count.
+    ///                     Currently only mono and stereo are
+    ///                     supported.
+    ///
```

Comment:
Even though it seems obvious, you could maybe add the information in parentheses:
"Currently only mono (1) and stereo (2) are supported"